src/components/buttons/pause-button.ts

```diff
@@ -1,12 +1,16 @@
-import { PAUSE_BUTTON } from "@constants";
-
+import { PAUSE_BTN_IMG } from '@constants';
```

Comment:
lets use double quotes just to be consistent for imports

Reply:
Changes applied, thank you.